assets/section-main-product.css

```diff
@@ -648,6 +681,40 @@ a.product__text {
     right: 5rem;
     top: 2.2rem;
   }
+  .image_label {
+    box-sizing: border-box;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+    padding: 4px 8px;
+    gap: 10px;
+    width: 174px;
+    height: 26px;
+    background: rgba(15, 30, 59, 0.03);
+    border: 1px solid rgba(15, 30, 59, 0.2);
+    border-radius: 6px;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+    position: relative;
+    bottom: 32px;
+    left: 8px;
+    z-index: 2;
+  }
+  
+  #image_type {
```

Comment:
Please look into using more specific combined selectors to avoid any possible conflicts between selectors, improve performance, and to help in understanding where specific styles belong.
See https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Selectors/Combinators

Example could be `.product__media-item > . image_label` (but think through this yourself too)

Reply:
Changed!